src/commonComponents/sidebar/Sidebar.js

```diff
@@ -0,0 +1,84 @@
+// react
```

Comment:
onclicking the last category in the preview it is scrolling up and the categories become invisible, can we fix that sidebar so that it doesnt scroll up

Reply:
Ya I've noticed this, didn't find a solution yet. I'll try to fix it

Reply:
resolved!!